prisma/schema.prisma

```diff
@@ -18,26 +18,28 @@ model User {
 }
 
 model Character {
-  user   User   @relation(fields: [id], references: [id])
-  id     Int    @unique
-  colour String @default("#FFFFFF")
-  bobux  Int    @default(0)
-  bio        String?
-  worlds     World[]
+  user     User       @relation(fields: [id], references: [id])
+  id       Int        @id
+  colour   String     @default("#FFFFFF")
+  bobux    Int        @default(0)
+  bio      String?
+  worlds   World[]
+  Visitors Visitors[]
```

Comment:
Actually Daniel, that field is autogenerated by the Prisma VSCode extension, and I cannot change it.

Reply:
Ah... and the reason why I couldn't change it was because it was necessary. The naming confused me to the point where I didn't even understand why the extension was generating this field.